src/types/eventsType.ts

```diff
@@ -0,0 +1,16 @@
+export type Event = {
+    id: string;
+    creator: string;
+    title: string;
+    slug: string;
+    description: string;
+    startTime: Date;
+    endTime: Date;
+    location: string;
+    link: string;
+    public: boolean;
+    lastEditTime: Date;
+    photo: string;
+    tags: string[];
```

Comment:
this is 100% my bad but the type here isn't actually `string[]`. (You can't tell coz the backend just returns an empty array for all the event tags). For now can you change this to `tags: unknown[];`. Thanks :)))